libr/include/r_util/r_print.h

```diff
@@ -192,6 +192,7 @@ R_API void r_print_offset_sg(RPrint *p, ut64 off, int invert, int offseg, int se
 #define R_PRINT_STRING_WIDE32 16
 #define R_PRINT_STRING_ESC_NL 32
 R_API int r_print_string(RPrint *p, ut64 seek, const ut8 *str, int len, int options);
+R_API int r_print_gameboy_string(RPrint *p, ut64 seek, const ut8 *buf, int len, int options);
```

Comment:
imho this shouldnt be a different function. We can hold the selected encoder inside RPrint and just call the decoder function before printing it in `r_print_string`.

The `pse` command can be replaced with a config variable that works globally. For temporal testing encodings we can use the `ps@e:cfg.encoding=pokemon`. (im proposing cfg.encoding, but it can be anything else